backstop/tests/table.html

```diff
@@ -92,46 +92,64 @@
     ></script>
 
     <script>
-      customElements.define('svg-folder', class extends HTMLElement {
-        connectedCallback() {
-          this.innerHTML = `<svg viewBox="0 0 16 16"><path d="M8,3,6.5856,1.5857A2,2,0,0,0,5.1715,1H1A1,1,0,0,0,0,2V5H16V4a1,1,0,0,0-1-1Z" fill="#e7b93a" /><path d="M0,14a1,1,0,0,0,1,1H15a1,1,0,0,0,1-1V5H0Z" fill="#f6d97a" /></svg>`;
+      customElements.define(
+        'svg-folder',
```

Comment:
oh wow, formatter was not run before? :O

Reply:
I think the vscode formatter is ok with both formats.

I removed the extra div in paginator using regex find and search, and it had a bunch of empty lines which the vscode formatter was not removing, so I had to run prettier which caused all these other changes